docs/_docs/customizingyourgateway.md

```diff
@@ -303,3 +303,17 @@ If no custom handler is provided, the default stream error handler
 will include any gRPC error attributes (code, message, detail messages),
 if the error being reported includes them. If the error does not have
 these attributes, a gRPC code of `Unknown` (2) is reported.
+
+## Routing Error handler
+To override how the error handling for cases when `*runtime.ServeMux` was not 
+able to serve due to routing issues, use the `runtime.WithRoutingErrorHandler` option. 
+
+This will configure all HTTP routing errors to pass through this error handler.
+Default behaviour is to map http error codes to gRPC errors.
```

Comment:
```suggestion
This will configure all HTTP routing errors to pass through this error handler.
Default behavior is to map HTTP error codes to gRPC errors.
```